src/indexables/application/commands/verify-non-timestamp-indexables-command.php

```diff
@@ -0,0 +1,42 @@
+<?php
+// phpcs:disable Yoast.NamingConventions.NamespaceName.TooLong -- Given it's a very specific case.
+namespace Yoast\WP\SEO\Indexables\Application\Commands;
+
+use Yoast\WP\SEO\Indexables\Domain\Abstract_Indexables_Command;
+use Yoast\WP\SEO\Indexables\Domain\Batch_Size;
+use Yoast\WP\SEO\Indexables\Domain\Current_Verification_Action;
+
+/**
+ * The Verify_Non_Timestamp_Indexables_Command class.
+ */
+class Verify_Non_Timestamp_Indexables_Command extends Abstract_Indexables_Command {
+
+	/**
+	 * The current verification action in progress.
+	 *
+	 * @var Current_Verification_Action $current_action
+	 */
+	private $current_action;
+
+	/**
+	 * The constructor.
+	 *
+	 * @param int    $last_batch            The last batch count.
+	 * @param int    $batch_size            The batch size.
+	 * @param string $current_action        The current verification action.
+	 */
+	public function __construct( int $last_batch, int $batch_size, string $current_action ) {
+		$this->current_action = new Current_Verification_Action( $current_action );
+
+		parent::__construct( $batch_size, $last_batch );
```

Comment:
I did this because the parent constructor sets these values. In the case we want to do some sort of validation before setting them it only needs to be applied in the parent :)